Gemfile

```diff
@@ -218,11 +218,11 @@ end
 unless ENV["APPLIANCE"]
   group :development do
     gem "foreman"
-    gem "haml_lint",        "~>0.20.0", :require => false
-    gem "rubocop",          "~>0.52.1", :require => false
+    gem "haml_lint",        "~>0.20.0",            :require => false
+    gem "rubocop",          ">=0.52.1", "<0.53.0", :require => false
```

Comment:
What's wrong with `~>0.52.1`?

Reply:
> It will handle the major version only, so 0.x.x.

No, `~>` only allows the last specified part to float.  So with x.y.z, x and y are fixed and z is the minimum.

Reply:
@bdunne Hum... you're right. I read [the documentation](http://bundler.io/v1.5/gemfile.html#gemfiles) again and it is exactly as you said. For some reason Scrutinizer is using Rubocop 0.55. I do not have access to that, but I will try to figure out if there is another solution. In the meantime, is it a problem to update the rules to use `0.53.x`?

Reply:
I'm not opposed to updating it as long as the config files and requirements are updated in all of the repos.